doc/read-the-docs-site/architecture/runtime-view.rst

```diff
@@ -0,0 +1,156 @@
+The chain-sync protocol essentially streams events to the client which have the following structure:
+
+.. code-block:: haskell
+
+  data ChainSyncEvent a
+    = RollForward a C.ChainTip
+    | RollBackward C.ChainPoint C.ChainTip
+    deriving (Show, Functor, Generic)
+
+where ``a`` can be anything including a block.
+
+Now, let's assume that we want to run multiple indexers that process inputs at the same rate, and that a subset of indexers are queryable from the a client application.
+Running multiple indexers that process inputs at the same rate are managed by a ``Coordinator``.
+The runtime view would look like:
+
+.. uml::
+
+    !include <C4/C4_Dynamic>
+    title Marconi indexing runtime view with multiple indexers
+
+    Container_Ext(datasource, "Datasource")
+    Container_Ext(dApp, "dApp")
+
+    Container_Boundary(marconiContainer, "Marconi application") {
+      Component(sourceReader, "Input source reader", "Haskell component", "Module that processes data from the external datasource and transforms it to a [ProcessedInput]")
+      Component(coordinator, "Coordinator", "Haskell component", "Combines multiple indexers so that they can process inputs at the same rate")
+      Component(indexerA, "Indexer A", "Haskell component")
+      Component(indexerB, "Indexer B", "Haskell component")
+      Component(indexerC, "Indexer C", "Haskell component")
+      Component(indexerD, "Read-only indexer D", "Haskell component", "Read-only indexer which doesn't store anything, but aggregates results of multiple indexers.")
+    }
+
+    Rel(datasource, sourceReader, "sends some input data")
+    Rel(sourceReader, coordinator, "sends input")
+    Rel(coordinator, indexerA, "forwards input")
+    Rel(coordinator, indexerB, "forwards input")
+    Rel(coordinator, indexerC, "forwards input")
+    Rel(indexerD, indexerA, "queries")
+    Rel(indexerD, indexerB, "queries")
+
+    Rel(dApp, indexerC, "queries")
+    Rel(dApp, indexerD, "queries")
+
+In this view, we show that the ``Coordinator`` can combine multiple indexers.
+The ``Coordinator``'s role is to foward the incoming inputs to each of the indexer and make sure each indexer has processed the input before moving to the next one.
+Additionnaly, some indexer can be "read-only" as they won't index any information, but can provide queries that aggregate the query results of other indexers.
+
+User indexer querying view
+--------------------------
+
+In this section, we'll show the querying runtime view of a Marconi application which contains at least one indexer.
+The Marconi indexer querying is typically done in a separate thread from the indexing itself.
+We're assuming that the indexers have fully consumed the input stream and are ready to be queried.
+
+.. note::
+
+   Although you can start querying the indexer even if it did not *fully* consumed the input stream.
+   The only issue is that the user will get out of date information when compared the actual tip of the local node.
+
+.. uml::
```

Comment:
Hmm, I think this part is missing from the webpage, at least for my local website. Could you double-check that it is appearing for you?